src/panels/lovelace/entity-rows/hui-timer-entity-row.ts

```diff
@@ -126,14 +126,17 @@ class HuiTimerEntityRow extends LitElement {
 
     if (stateObj.state === "idle" || this._timeRemaining === 0) {
       return (
-        this.hass!.localize(`state.timer.${stateObj.state}`) || stateObj.state
+        this.hass!.localize(`component.timer.state._.${stateObj.state}`) ||
```

Comment:
Made the change, but that lead me to something else: The more-info dialog also was not aligned, which I fixed now, but that meant duplicating code. But since the `computeStateDisplay()` function does only accept a `stateObj` we cannot pass the remaining seconds to make it completely capsuled in `computeStateDisplay()`.

We could add an optional parameter to `computeStateDisplay()` for a generic value to be provided by the caller and to be used in the domain specific handling for `timer` then.

Reply:
Isn't it the same as `stateObj.attributes.duration`?

Reply:
I don't think so. `stateObj.attributes.duration` is the defined runtime that this timer runs to reach 0. `timeRemaining` is the remaining seconds from the start of the timer until the duration is reached.

Reply:
Yeah I meant `stateObj.attributes.remaining`

Reply:
OK, let's move it to `data/timer` then

Reply:
`timerTimeRemaining` should move there too btw

Reply:
Code is moved. Have a look if that is what you had in mind.